src/Components/Wallet/Wallet.jsx

```diff
@@ -490,8 +490,6 @@ class Wallet extends Component {
         let v1Balance = helper.formatDbets(this.state.balances.oldToken.amount)
```

Comment:
This would have to be 
```suggestion
        let v1Balance = this.state.balances.oldToken.amount
```
since `helper.formatDbets` results in a floating point as well